go.mod

```diff
@@ -0,0 +1,197 @@
+module github.com/onflow/random-coin-toss
+
+go 1.19
+
+require (
+	github.com/bjartek/overflow v1.14.1
+	github.com/onflow/cadence v0.42.2
+	github.com/onflow/flow-go/crypto v0.24.9
```

Comment:
I made a recent PR to improve the statistical tests (https://github.com/onflow/flow-go/pull/4844). It can help reduce flakiness in the test. Maybe I can merge that PR and tag a new version to be used here first.

Reply:
Sounds good. I'll bump the version once it's ready to go.

Reply:
I just merged the PR and tagged a new version `v0.24.10` that you can use